### DISCUSSION TITLE
Standard adókód publikálásra került

### DISCUSSION THREAD

**Discussion by renced42** [nav-gov-hu/eVAT#18] [Közlemények]
Tájékoztatunk mindenkit, hogy az eÁfa rendszerhez kapcsolódóan a NAV standard adókód lista feltöltésre került a https://github.com/nav-gov-hu/eVAT/tree/master/docs/standard_adokod alá.

Várjuk szíves észrevételeiket és véleményüket!

**Comment by bmatyi94** [comment]
Olyan kérdésem lenne, hogy az excel Q oszlopában lévő Kulcs (K) oszlop értékei (VR, TEV,0) mit jelent? Valamint ezen kulcsokra miért van szükség?

**Reply by bmatyi94** [reply]
@renced42

**Comment by pahitamas** [comment]
nekem nem működik a link "404 This is not the web page you are looking for" hibát kapok.
